util/perl/OpenSSL/Test.pm

```diff
@@ -831,6 +832,49 @@ sub openssl_versions {
     return @versions;
 }
 
+=over 4
+
+=item B<ok_nofips test>
+
+Returns ok(test) if the environment variable FIPS_MODE is undefined,
+otherwise it returns ok(!test). This can be used for ok tests that will
+fail in FIPS mode only.
+
+An example:
+
+  ok_nofips(run(app(["md5.pl"])));
+
+=item B<is_nofips test>
+
+Returns is(test) if the environment variable FIPS_MODE is undefined,
+otherwise it returns is(test). This can be used for is tests that will
+fail in FIPS mode only.
+
+=item B<isnt_nofips test>
```

Comment:
Likewise here

Reply:
The notation inside test::more is not so elaborate. Hopefully this is better now.

Reply:
The notation in Test::More is radically *different*:

> ```
>        ok
>              ok($got eq $expected, $test_name);
> ...
>        is
>        isnt
>              is  ( $got, $expected, $test_name );
>              isnt( $got, $expected, $test_name );
> ```